source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3174,6 +3174,26 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
    .. versionadded:: 2.1.0
 
 
+.. mixxx:control:: [Library],MoveTrackUp
+
+   Equivalent to pressing the :kbd:`Alt` + :kbd:`Up` key on the keyboard
```

Comment:
I'm not sure everyone browsing these controls is aware of the new Alt move feature.
I suggest to add more context here in order to make this feature more discoverable.
For example:
> Move the track selection up by one row. This is applicable only in (unlocked) Playlists.

Same for down and MoveTrack.

Reply:
I reworded a bit both texts, what do you think?